MeetupPlatformApi/DataTransferObjects/MeetupInputDto.cs

```diff
@@ -1,13 +1,25 @@
-﻿namespace MeetupPlatformApi.DataTransferObjects;
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MeetupPlatformApi.DataTransferObjects;
 
 public class MeetupInputDto
 {
+    [Required]
+    [MaxLength(30)]
+    [DataType(DataType.Text)]
```

Comment:
Please, get rid of the `[DataType]` annotations. Asp validator and Swagger are pretty good in detecting field types